policyengine_us/variables/gov/states/mt/tax/income/deductions/cdcc/mt_child_dependent_care_expense_deduction_eligible_children.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class mt_child_dependent_care_expense_deduction_eligible_children(Variable):
+    value_type = int
+    entity = TaxUnit
+    label = "Montana child dependent care expense deduction"
+    definition_period = YEAR
+    defined_for = StateCode.MT
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.mt.tax.income.deductions.standard.child_dependent_care_expense_deduction
+        person = tax_unit.members
+        age = person("age", period)
+        dependent = person("is_tax_unit_dependent", period)
+
```

Comment:
Also add a 15yo test (please always include tests at boundaries)
```suggestion
        age_eligible = age < p.age_threshold
        eligible = dependent & (age_eligible | person("is_disabled", period))
```

Reply:
you didn't change the `<=` to `<` or accept my restructuring suggestion. why did you resolve this?